src/pages/docs/game/chat.mdx

```diff
@@ -0,0 +1,20 @@
+---
+layout: ../../../layouts/docs.astro
+---
+
+
+# __Chat__ 
+[@veritanuda](https://keybase.io/veritanuda)
```

Comment:
I'm not sure yet if adding user info to the actual core docs is useful information.

For the guides (e.g. the markdown tutorial) it feels more relevant to me as it's a bigger content piece that always has some subjective preferences of the author, whereas for the core docs they're ideally just facts about the core functionality.

If you feel strongly about being attributed though, which I wouldn't mind, we might want to add a contribution section for each document.

Reply:
Hmm I just copy and pasted from before that is why. But I get your point. I can remove it.